web/app/routes/authenticate.ts

```diff
@@ -1,15 +1,19 @@
 import Route from "@ember/routing/route";
 import { inject as service } from "@ember/service";
+import ConfigService from "hermes/services/config";
 import SessionService from "hermes/services/session";
 
 export default class AuthenticateRoute extends Route {
+  @service("config") declare configSvc: ConfigService;
   @service declare session: SessionService;
 
   beforeModel() {
     /**
-     * Checks if the session is authenticated,
+     * If using Google auth, checks if the session is authenticated,
      * and if it is, transitions to the specified route
      */
-    this.session.prohibitAuthentication("/");
+    if (!this.configSvc.config.skip_google_auth) {
+      this.session.prohibitAuthentication("/");
```

Comment:
What happens if remove this `if` statement and run `prohibitAuthentication` under `skipGoogleAuth` conditions?

Reply:
Looks like it's the same behavior. I think I forgot what this method actually does - I can revert this change.

Reply:
It's supposed to redirect you to `/` if you're authenticated with Ember Simple Auth

Reply:
Yeah, actually I guess we should probably just redirect right away from this route if we're bypassing Google auth... Because users could still have this URL saved and it won't be relevant without Google auth.

Reply:
https://github.com/hashicorp-forge/hermes/pull/144/commits/8d11b85038a6155bb28efc9c6ea4c6853272a247 implements this ^